spec/system/search_spec.rb

```diff
@@ -27,4 +26,12 @@
 
     expect(page).to have_css '.agency-separator'
   end
+
+  scenario "paging doesn't break js" do
+    visit "/"
+    find_all("nav.pagination").first.find_all(".page")[1].click
+    sleep 1
+    # gov banner should remain closed
+    expect(find("#gov-banner").visible?).to be_falsey
```

Comment:
The test to see if it works:
1. click on a pagination button
2. see if the top uswds is closed like it should be